src/wp-includes/class-wp-theme.php

```diff
@@ -1714,6 +1733,46 @@ public static function get_allowed_on_site( $blog_id = null ) {
 		return (array) apply_filters( 'site_allowed_themes', $allowed_themes[ $blog_id ], $blog_id );
 	}
 
+	/**
+	 * Returns the folder names of the block template directories.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param bool $default Optional. Whether to return the default values. Default false.
+	 * @return string[] {
+	 *     Folder names used by block themes.
+	 *
+	 *     @type string $wp_template      Theme-relative directory name for block templates.
+	 *     @type string $wp_template_part Theme-relative directory name for block template parts.
+	 * }
+	 */
+	public function get_block_template_folders( $default = false ) {
+		$default_template_folders = array(
+			'wp_template'      => 'templates',
+			'wp_template_part' => 'parts',
+		);
```

Comment:
We can use a class property here instead of creating a new variable everytime this method is run.

Reply:
Done.